lib/templates/md/top_level_property.md

```diff
@@ -7,10 +7,11 @@
 {{>categorization}}
 
 {{#hasNoGetterSetter}}
-{{{ linkedReturnType }}} {{>name_summary}}
+{{{ linkedReturnType }}} {{>name_summary}}  {{!two spaces intentional}}
```

Comment:
What is the purpose of "two spaces intentional"?

Reply:
Two spaces at the end of a line renders a line break, but not a paragraph break.

Normally a line break just renders a space (unless there's other formatting involved, like headings). Two line breaks renders a paragraph break, separating the blocks with extra vertical space that I didn't want.